src/server/database/setupDB.js

```diff
@@ -1,48 +0,0 @@
-import r from './rethinkdriver';
-
-const databases = ['action', 'actionTesting'];
-
-const database = [
```

Comment:
Unspecified for now. Was considering adding a sequence to the `npm run test` that would create a test db via migrations, run tests, then destroy db...but figure we can cross that bridge in a bit (a few weeks maybe?)